al_aws_collector.js

```diff
@@ -73,7 +74,7 @@ class AlAwsCollector {
         })
     }
     
-    constructor(context, collectorType, ingestType, version, aimsCreds, formatFun) {
+    constructor(context, collectorType, ingestType, version, aimsCreds, formatFun, healthCheckFuns) {
```

Comment:
May I ask you please to fix class declaration header here https://github.com/alertlogic/al-aws-collector-js/pull/7/files#diff-721ff284e807cc07d540aa9120b6277eR48